docs/modules/integrate/pages/feast-config.adoc

```diff
@@ -0,0 +1,70 @@
+= Configure Feature Store
+:page-enterprise: true
+:description: To use a Feast project, you must configure the feature store. The configuration is defined in a YAML configuration file.
+
+{description}
+
+The _feature_store.yaml_ file defines the following:
+
+* The environment in which Feast deploys and operates. This is defined in the `project` section
+* The location of the feature registry. This is defined in the `registry` section
+* The online store. This is defined in the `online_store` section
+* The offline store. This is defined in the `offline_store` section
+* The namespace to use for the feature store. This can be used to isolate multiple deployments in a single Feast installation, and can contain only letters, numbers, and underscores. This is defined in the `project` section
+* The materialization engine. This is defined in the `engine` section
+* The serialization service. This serializes the entity key to a bytestring to allow it to be be used as a lookup key in a hash table. You can specify Feast's serialization service. This is defined in the `entity_key_serialization_version` section
+
+== Configure Hazelcast as Online Store
+
+The `online-store` section configures the online store and initializes a config object, which is passed to the `OnlineStore` interface by Feast to create the connection.
+
+The available options are as follows:
+
+[cols="1,1,1,1"]
+|===
+|Name |Type |Description |Default Value
+
+|type
+|String
+|Online store type selector
+|"hazelcast"
+
+|TBD
+|TBD
+|TBD
+|TBD
+|===
+
+=== Example Local Cluster Configuration
+
+You can configure the connection to a local cluster as shown in the following example:
+
+[source,yaml]
+----
+online_store:
+    type: hazelcast
+    cluster_name: <YOUR_CLUSTER_NAME>
+    cluster_members: [<YOUR_CLUSTER_MEMBERS>]
+    creds_username: <YOUR_CRED_USERNAME>
+    creds_password: <YOUR_CRED_PASSWORD>
+    ca_path: <YOUR_CA_PATH_HERE>
+    cert_path: <YOUR_CERT_PATH_HERE>
+    key_path: <YOUR_KEY_PATH_HERE>
+    ssl_password: <YOUR_SSL_PASSWORD_HERE>
+----
+
+=== Example SSL-enabled Cluster Configuration
+
+You can configure an SSL-enabled connection as shown in the following example:
+
+[source,yaml]
+----
+online_store:
+    type: hazelcast
+    cluster_name: <YOUR_CLUSTER_NAME>
+    ca_path: <YOUR_CA_PATH_HERE>
+    cert_path: <YOUR_CERT_PATH_HERE>
+    key_path: <YOUR_KEY_PATH_HERE>
+    ssl_password: <YOUR_SSL_PASSWORD_HERE>
```

Comment:
This section needs an update once the configuration keys in the following section are updated.